CONTRIBUTING.md

```diff
@@ -0,0 +1,15 @@
+# Reporting a bug
```

Comment:
Bitte jedenfalls auch erweitern um die möglichkeit einer Frage oder eines Problems in der Anwendung des Beispielcodes.
Das gilt für die ganze Formulierung: Wir stellen hier Codebeispiele für die Integration bereit, d.h. wir sollten auch bei den Issues die guidelines so anpassen.
Aus meienr Sicht gibt es 5 Kategorien:
* Fragen
* Anfragen um weitere Beispielszenarien (andere Programmiersprachen, weitere Sezanrien)
* Probleme: Der Code funktiuoniert nicht (entweder aufgrund eines Fehlers im sample code oder aufgrund fehlerhafter Konfiguration oder adaptierung)
* Security issues
* eigene Beispiele, die diese Beispielsammlung erweitern

Für alle fünf sollten wir kurz und prägnant formuleiren was wir brauchen